### PR TITLE
Purge expired auth cookies

### LINKED ISSUE #4982: Purge expired AuthCookies
## Description of problem

It appears AuthCookies stay in the database forever, leading some users to see millions of records.

## Description of solution

There is a [DeleteExpiredCookies on AuthCookieController](https://github.com/dnnsoftware/Dnn.Platform/blob/develop/DNN%20Platform/Library/Security/Cookies/AuthCookieController.cs#L58) but that doesn't appear to be used anywhere. It seems obvious we should call it from a scheduled task or similar.

### PR DESCRIPTION
**Pull request by donker**
## Summary
Add a scheduled task to purge AuthCookies from the DB that have expired.

Fixes #4982

### CODE REVIEW

**Review by sleupold — COMMENTED** [review]
sproc: Using TOP should include an "Order By" statement.

**Comment by donker** [comment]
> sproc: Using TOP should include an "Order By" statement.

Not for DELETE TOP.

**Review by valadas — APPROVED** [review]
Looks good to me, seen a ton of delete examples that don't have an OrderBy

File: DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.11.00.SqlDataProvider [hunk 1]
```diff
@@ -24,6 +24,35 @@ WHERE SettingName = 'SSLSetup'
  AND SettingValue = 'True'
 GO
 
+
+/*    Add scheduled task to delete soft deleted users       */
```
Comment (bdukes) [inline DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.11.00.SqlDataProvider:28]:
```suggestion
/*    Add scheduled task to delete expired auth cookies       */
```

**Review by bdukes — APPROVED** [review]

**Review by mitchelsellers — APPROVED** [review]

**Comment by sleupold** [comment]
for large tables, it might be useful to add an index on "ExpiresOn" column and use "ORDER BY ExpiresOn DESC"

**Comment by WillStrohl** [comment]
Nice one! :)

**Pull request merged**